openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -129,33 +134,115 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
   }
 
   .title--sub {
+    @extend %hd-4;
     color: $heading-staff-color;
     margin-top: ($baseline-v/2);
     margin-bottom: ($baseline-v/2);
   }
 
   .student__answer__display__content {
-    border: 1px solid rgba($color-decorative-staff, 0.25);
-    @include padding(($baseline-v/2), ($baseline-h/2), ($baseline-v/2), ($baseline-h/2));
+    border: 1px solid rgba($copy-staff-color, 0.25);
+    padding: ($baseline-v/2) ($baseline-h/4);
     margin-bottom: ($baseline-v/2);
   }
 
-  .openassessment__student-info_list {
+  .staff-info__student__report {
     list-style-type: none;
+
+    .title {
+      @extend %t-strong;
+      margin-top: ($baseline-v/2);
+      border-top: 1px solid $heading-staff-color;
+      padding: ($baseline-v/2) ($baseline-h/2) ($baseline-v/2) 0;
+
+      span {
+        font-weight: inherit;
+      }
+    }
+  }
+
+  .staff-info__cancel-submission__content,
+  .staff-info__staff-override__content {
+    padding: $baseline-v ($baseline-h/2);
+    background-color: white;
   }
 
   .value {
     width: $max-width/2;
   }
 
-  /**
-   * The follow styles are bound for the "shame" file. This is done to override
-   * LMS specific styles on HTML elements.
-   */
+  // staff assessments
+  .wrapper--staff-assessment {
+    margin-top: ($baseline-v/2);
```

Comment:
I should have said that I have the same concern that the sass is getting out of control, and that there are a lot of changes in the _developer.scss. I don't know the best time to try to sort that out.